ocaml/xapi/xapi_vm_clone.ml

```diff
@@ -367,6 +367,7 @@ let copy_vm_record ?snapshot_info_record ~__context ~vm ~disk_op ~new_name
     ~vCPUs_max:all.Db_actions.vM_VCPUs_max
     ~vCPUs_at_startup:all.Db_actions.vM_VCPUs_at_startup
     ~vCPUs_params:all.Db_actions.vM_VCPUs_params
+    ~actions_after_softreboot:`soft_reboot
```

Comment:
This still has the wrong value.

Reply:
`soft_reboot is the default value, what value would it need?

Reply:
This is a copy; we don't want the default. Use `all.Db_actions.vM_actions_after_softreboot` (see the other ones).